hbase-client/src/main/java/org/apache/hadoop/hbase/client/metrics/ScanMetricsUtil.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.client.metrics;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+import org.apache.yetus.audience.InterfaceAudience;
+
+@InterfaceAudience.Private
+public final class ScanMetricsUtil {
+
+  private ScanMetricsUtil() {
+  }
+
+  /**
+   * Creates a new counter with the specified name and stores it in the counters map.
+   * @return {@link AtomicLong} instance for the counter with counterName
+   */
+  static AtomicLong createCounter(Map<String, AtomicLong> counters, String counterName) {
```

Comment:
This is sort of mix of comments and Javadoc, let's convert all method comments into Javadocs